### PR TITLE
Fix: qa fixes

### PR DESCRIPTION
**Pull request by denviljclarke**
### Description

This PR made the final design QA tweaks based on [this document](https://docs.google.com/document/d/1SzuW-NfZedxpIcuP3UQzT6LJ0WkDv_86hT5uxd10oik/edit)

### CODE REVIEW

**Review by olenovyk — APPROVED** [review]
LGTM!

File: src/components/Amount.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ export default function Amount({
   }
```
Comment (chapati23) [inline src/components/Amount.tsx:32]: image

font too small, should be 26px not 22px

File: src/components/Amount.tsx [hunk 2]
```diff
@@ -32,7 +32,7 @@ export default function Amount({
...
```
Comment (chapati23) [inline src/components/Amount.tsx:33]: Why is cKES behind eXOF although it has more outstanding supply? everything else is ordered by supply

image

**Review by philbow61 — APPROVED** [review]

File: src/components/Amount.tsx [hunk 3]
```diff
@@ -32,7 +32,7 @@ export default function Amount({
...
 
   return (
```
Comment (chapati23) [inline src/components/Amount.tsx:34]: collat ratio number should be vertically centered (left current code, right side design)

image

File: src/components/Amount.tsx [hunk 4]
```diff
@@ -32,7 +32,7 @@ export default function Amount({
...
-    <CardBackground className="flex flex-col gap-2 px-6 py-4 md:min-w-[220px] md:gap-4">
```
Comment (chapati23) [inline src/components/Amount.tsx:35]: Font size too small, should be 26px but was 22px

image

File: src/components/Amount.tsx [hunk 5]
```diff
@@ -32,7 +32,7 @@ export default function Amount({
...
+    <CardBackground className="flex flex-col gap-2 px-6 py-4 md:min-w-[220px] md:gap-4 ">
```
Comment (chapati23) [inline src/components/Amount.tsx:35]: tile spacing is wrong (left code, right design), should have more space between grid elements

image

File: src/components/Amount.tsx [hunk 6]
```diff
@@ -32,7 +32,7 @@ export default function Amount({
...
       <div className="flex items-center">
```
Comment (chapati23) [inline src/components/Amount.tsx:36]: Font size too small, should be 22px but was 18px

image

File: src/components/Amount.tsx [hunk 7]
```diff
@@ -32,7 +32,7 @@ export default function Amount({
...
         {iconSrc && (
```
Comment (chapati23) [inline src/components/Amount.tsx:37]: too much spacing

image

File: src/components/Amount.tsx (hunk 2 above)
Comment (denviljclarke) [reply src/components/Amount.tsx:33]: It's just a coincidence. There's no sorting. I've added sorting, it wasn't specified previously or noted by Roman or Ole in any of the past several reviews

Comment (denviljclarke) [reply src/components/Amount.tsx:33]: They're in release order

**Pull request merged**